src/frontend/src/components/RecordSearch/Record/Case.tsx

```diff
@@ -19,6 +19,7 @@ export default class Cases extends React.Component<Props> {
       location,
       current_status,
     } = this.props.case;
+    let allIneligible = charges.every(charge => charge.expungement_result.type_eligibility.status === "Ineligible");
```

Comment:
this can be `const` instead of `let`. Looks good otherwise, so I'm approving and you can just merge after you make that change.